pyproject.toml

```diff
@@ -25,20 +25,29 @@ line-length = 120
 
 [tool.ruff.lint]
 select = [
+  "A",       # flake8-builtins
+  # "ANN",   # flake8-annotations
```

Comment:
This one is good, but it requires all arguments to be type annotated. We're not ready yet.